executor/simple.go

```diff
@@ -136,6 +137,10 @@ func (e *SimpleExec) executeRollback(s *ast.RollbackStmt) error {
 	return nil
 }
 
+type sqlExec interface {
```

Comment:
>  sqlExec interface can be extract to sqlexec package, but I'll leave it in another tiny refact.

I said I'll refact it later.

This interface is needed because `sqlexec.ExecRestrictedSQL` can't satisfy our demand, I need something that commit the SQL after exec.